### ISSUE TITLE
Lists in template arguments may be parsed as text

### ISSUE THREAD

**Issue by vroland** [vroland/mediawiki-peg-rust#3]
```
{{todo|
*Erwähnen: nun können wir ganz normal rechnen, wie wir es wollten (es ist 
genau so definiert, dass das funktioniert
*Abschnitt überarbeiten}}
```

is a list, 

```
{{todo|*Erwähnen: nun können wir ganz normal rechnen, wie wir es wollten (es ist 
genau so definiert, dass das funktioniert
*Abschnitt überarbeiten}}
```

is not.

**Comment by kulla** [comment]
Sitenote: Parsing content of `{{todo|...}}` templates is not very important

**Comment by vroland** [comment]
This applied to templates in general, not just `todo`.
fixed in d349e8daac784b21bfb351339c8d34c9f1f43b72.
